libautoscoper/src/Tracker.cpp

```diff
@@ -613,6 +641,83 @@ std::vector <double> Tracker::trackFrame(unsigned int volumeID, double* xyzypr)
       views_[i]->radRenderer()->set_viewport(viewport[0], viewport[1],
         viewport[2], viewport[3]);
 
+      // DRR projection
+      // Performed by the RayCaster class
+      // called by the Tracker::trackFrame() method
+      // @ = DRR pixel with some intensity value
+      // -------------------------------
+      // |                             |
+      // |             ^               |
+      // |            /@\              |
+      // |           /@@@\             |
+      // |          /@@@@@\            |
+      // |          \@@@@@/            |
+      // |           \@@@/             |
+      // |            \@/              |
+      // |             v               |
+      // |                             |
+      // -------------------------------
+      //
+      // Crop the radiograph to the bounds of the DRR
+      // Performed by the RadiographRenderer class
+      // called by the Tracker::trackFrame() method
+      // ! = some pixel with some intensity value, inside the region of interest
+      // $ = some pixel with some intensity value, outside the region of interest
+      // -------------------------------
+      // |$$$$$$$$$$$$$$$$$$$$$$$$$$$$$|
+      // |$$$$$$$$$$$$$^$$$$$$$$$$$$$$$|
+      // |$$$$$$$$$$$$/!\$$$$$$$$$$$$$$|
+      // |$$$$$$$$$$$/!!!\$$$$$$$$$$$$$|
+      // |$$$$$$$$$$/!!!!!\$$$$$$$$$$$$|
+      // |$$$$$$$$$$\!!!!!/$$$$$$$$$$$$|
+      // |$$$$$$$$$$$\!!!/$$$$$$$$$$$$$|
+      // |$$$$$$$$$$$$\!/$$$$$$$$$$$$$$|
+      // |$$$$$$$$$$$$$v$$$$$$$$$$$$$$$|
+      // |$$$$$$$$$$$$$$$$$$$$$$$$$$$$$|
+      // -------------------------------
+      //
```

Comment:
// $ = some pixel with some intensity value, inside the region of interest
      // & = some pixel with some intensity value, outside the region of interest
      // -------------------------------
      // |&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&|
      // |&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&|
      // |&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&|
      // |&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&|
      // |&&&&&&&&$$$$$$$$$$$$$$$$$$$$$$$$$$$$&&&|
      // |&&&&&&&&$$$$$$$$$$$$$^$$$$$$$$$$$$$$&&&|
      // |&&&&&&&&$$$$$$$$$$$$/$\$$$$$$$$$$$$$&&&|
      // |&&&&&&&&$$$$$$$$$$$/$$$\$$$$$$$$$$$$&&&|
      // |&&&&&&&&$$$$$$$$$$/$$$$$\$$$$$$$$$$$&&&|
      // |&&&&&&&&$$$$$$$$$$\$$$$$/$$$$$$$$$$$&&&|
      // |&&&&&&&&$$$$$$$$$$$\$$$/$$$$$$$$$$$$&&&|
      // |&&&&&&&&$$$$$$$$$$$$\$/$$$$$$$$$$$$$&&&|
      // |&&&&&&&&$$$$$$$$$$$$$v$$$$$$$$$$$$$$&&&|
      // |&&&&&&&&$$$$$$$$$$$$$$$$$$$$$$$$$$$$&&&|
      // -------------------------------
      //